models/location-sql.js

```diff
@@ -0,0 +1,58 @@
+import db from '../db/connection.js'
+
+// Get all location data
+export async function getAllLocationData() {
+    const result = await db.query(`SELECT * FROM location;`);
+    return result.rows;
```

Comment:
in media-sql.js we use "RETURNING *" at the end of the SQL query
this allows us to simply return the result rather than result.rows